### PR TITLE
fixes #127: ability to disable desktop notifications

### LINKED ISSUE #127: Option to disable desktop notifications
Hi @codegangsta!

Could you please add ability to disable desktop notifications? While the feature seems to be helpful and nice maybe some people wouldn't like it. Personally I don't like such notifications on my screen and I would like to disable it via CLI flags/parameters.

screenshot_2017-10-25_12-36-16

Anyway thank you for your project! ;-)

### PR DESCRIPTION
**Pull request by resurtm**
Please check details at #127

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Options
    --certFile value              TLS Certificate
    --keyFile value               TLS Certificate Key
    --logPrefix value             Setup custom log prefix
+   --disableNotifications        disable desktop notifications
```
Comment (ibash) [inline README.md]: @resurtm can we actually make desktop notifications disabled by default (and add a flag to enable them)?

**Comment by ibash** [comment]
@resurtm other than reversing the default behavior, I'd be happy to merge this :)

**Comment by resurtm** [comment]
@ibash Thanks! Done.

**Comment by glutamatt** [comment]
I built your branch : thanks dude, you saved my notification stack ! :+1:

**Pull request merged**